### PR TITLE
Remove landing page

### LINKED ISSUE #1465: Introduce a flag to disable the default home page
Multiple security audits got triggered on our company because they found the windows exporter service responded with an html page at the root and exposed the endpoint from there (`/metrics`, `/health`, `/version`)

Even though I’m not a big fan of security by obfuscation maybe introducing a flag to disable the home page (`/`) could make sense and make the exporter less visible in enterprise environment.

### PR DESCRIPTION
**Pull request by jkroepke**
Fixes #1465

### CODE REVIEW

File: exporter.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ func main() {
 			"web.disable-exporter-metrics",
 			"Exclude metrics about the exporter itself (promhttp_*, process_*, go_*).",
 		).Bool()
+		disableExporterLadingPage = app.Flag(
```
Comment (breed808) [inline exporter.go:67]:
```suggestion
		disableExporterLandingPage = app.Flag(
```

**Review by breed808 — COMMENTED** [review]
Given `node_exporter` exposes a similar landing page by default, my preference would be to have this flag default to `true`. Thoughts?

**Comment by jkroepke** [comment]
> my preference would be to have this flag default to `true`.

I guess then, we can remove the entire feature. Who will set this flag to true?

**Comment by breed808** [comment]
I was thinking that too. Let's remove it and we can always revert if there's negative feedback regarding the removal.

**Review by breed808 — APPROVED** [review]
:+1:

**Pull request merged**